### DISCUSSION TITLE
Cant delete topics after strimzi version upgrade

### DISCUSSION THREAD

**Discussion by Yedidyah27** [strimzi/strimzi-kafka-operator#4840] [Q&A]
Hey,
I have upgraded my kafka operator to 0.22.1 and I now cant seem able to delete topics through the topic operator. I think that I have might have deleted the __strimzi_store_topic and __strimzi-topic-operator-kstreams-topic-store-changelog topics
 before the upgrade. Now that they are created I am getting the following message when trying to delete a topic:
`Failure processing KafkaTopic watch event DELETED on resource ha-gmw1r-ijsgfgfgfgf9q-1454545454---96f91a172718c0c54bf179672a6ab8a3ba2d2921 with labels {strimzi.io/cluster=aa-kafka-connectors1-st}: The state store, topic-store, may have migrated to another instance.`
Has anyone encountered something similar and has a solution?

**Comment by scholzj** [comment]
@alesj @tombentley @sknot-rh Any idea?

**Reply by alesj** [reply]
> The state store, topic-store, may have migrated to another instance.

This does look like something Kafka Streams would report ...
But since we have just one instance of TO/Streams, it cannot migrate to another instance ... ?!
Maybe deletion of those topics confuses Streams?
